HousingRepairsOnlineApi/Helpers/IRepairBookingResponseHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using HousingRepairsOnlineApi.Domain;
+using Microsoft.Azure.Cosmos;
+
+namespace HousingRepairsOnlineApi.Helpers
+{
+    public interface IRepairBookingResponseHelper
+    {
+        dynamic GetRepairBookingResponse(Repair repair);
```

Comment:
The use of `dynamic` feels like a code smell to me.
Is it not possible to have a specific type (either an interface or base class)?